### ISSUE TITLE
Tool Change - Resume XYZ move

### ISSUE THREAD

**Issue by micksulley** [vlachoudis/bCNC#952]
Using the tool change routine, at tool change it moves the gantry to the tool change position, then pressing Resume moves it to the probe position, then after probing it moves back to the tool change position.  All OK so far, but after pressing Resume again to start cutting again it moves back to the work, but moves X, Y and Z all at the same time.  In my view it would be much better if it moved X and Y first and then Z when it is back at the work.
I can work around this by carefully setting my tool change position, but I think this would be a worthwhile improvement.

**Comment by Harvie** [comment]
Yes! It would surely make sense to first retract to safe Z, move XY and plunge Z again in all of these situations.
